### DISCUSSION TITLE
为啥要删除migrations呢?

### DISCUSSION THREAD

**Discussion by xuqianjin** [midwayjs/midway#3960] [Q&A]
### Describe the problem（描述问题）

![image](https://github.com/user-attachments/assets/850d7ff0-08fd-494d-9d30-f6de900b72bd)

### Midway Versions（Midway 版本）

3.16.2

**Comment by czy88840616** [comment]
migrations是给脚本用的，不是给正式执行用的。

**Reply by xuqianjin** [reply]
![image](https://github.com/user-attachments/assets/e7129ef5-684b-4371-af38-324309fa1cd1)

但是typeorm的migrationsRun功能是需要用到这个的...

**Reply by czy88840616** [reply]
我的意思是，migrations 一般都是外部脚本执行一次，这个总不会用 midway 来写吧
